Protocol.md

```diff
@@ -116,6 +124,7 @@ The sending node MUST:
   - set `total_collateral_satoshis` to a value greater than or equal to 1000.
   - set `contract_maturity_bound` and `contract_timeout` to either both be UNIX timestamps, or both be block heights as distinguished [here](https://en.bitcoin.it/wiki/NLockTime).
   - set `contract_maturity_bound` to be less than `contract_timeout`.
+  - use a unique `input_serial_id` for each input
```

Comment:
should there be a similar requirement for output ids?

Reply:
This is in the offer and there is only one id, so they can't have a duplicate

Reply:
Added the requirement for the accept message